pkg/controllers/proxy/service_endpoints_sync.go

```diff
@@ -305,7 +309,10 @@ func (nsc *NetworkServicesController) setupExternalIPServices(serviceInfoMap ser
 
 				// ensure VIP less director. we dont assign VIP to any interface
 				err = nsc.ln.ipAddrDel(dummyVipInterface, externalIP)
-
+				if err != nil {
+					glog.Errorf("Failed to delete external ip adress from dummyVipInterface due to %s", err)
+					continue
```

Comment:
Not sure if we want to continue here or return

Reply:
I think continuing here is correct. That allow any other VIPs that might be able to be setup to continue to be setup rather than just halting where it is.